Cargo.toml

```diff
@@ -115,38 +117,101 @@ crossbeam = "0.8"
 crossbeam-channel = "0.5"
 csv = "1.1.1"
 curve25519-dalek = "3"
+delay-detector = { path = "tools/delay-detector" }
 derive-enum-from-into = "0.1.1"
 derive_more = "0.99.9"
 dirs = "4"
+dynasm = "1.0"
+dynasmrt = "1.0"
 easy-ext = "0.2"
 ed25519-dalek = "1"
 elastic-array = "0.11"
 enum-map = "2.1.0"
+enumset = "1.0"
 expect-test = "1.3.0"
+finite-wasm = "0.3.0"
 flate2 = "1.0.22"
 fs2 = "0.4"
 futures = "0.3.5"
 futures-util = "0.3"
+genesis-populate = { path = "genesis-tools/genesis-populate"}
+hashbrown = "0.11"
 hex = { version = "0.4.2", features = ["serde"] }
+hex-literal = "0.2"
 hkdf = "0.12.3"
 hyper = { version = "0.14", features = ["full"] }
 hyper-tls = "0.5.0"
 im = "15"
+indexmap = "1.6"
 indicatif = { version = "0.15.0", features = ["with_rayon"] }
 insta = { version = "1.26.0", features = ["json", "yaml"] }
+integration-tests = { path = "integration-tests" }
 itertools = "0.10.0"
 itoa = "1.0"
 json_comments = "0.2.1"
+lazy_static = "1.4"
+leb128 = "0.2"
 libc = "0.2.81"
 libfuzzer-sys = { version = "0.4", features = ["arbitrary-derive"] }
 log = "0.4"
 loupe = "0.1"
 lru = "0.7.2"
+memmap2 = "0.5"
 memoffset = "0.6"
+more-asserts = "0.2"
+near-account-id = { path = "core/account-id", features = [ "internal_unstable" ] }
```

Comment:
This is pretty nice. Features should always be specified by crates that require them, though.

Reply:
For features, is it a general comment, or a suggested change? In this specific case I think enabling the `internal_unstable` feature actually makes sense, as any `.workspace = true` dependency on it would necessarily be internal.